pkg/inst/tests/test_rdd.R

```diff
@@ -314,6 +314,18 @@ test_that("values() on RDDs", {
   expect_equal(actual, lapply(intPairs, function(x) { x[[2]] }))
 })
 
+test_that("pipeRDD() on RDDs", {
+  actual <- collect(pipeRDD(rdd, "cat"))
```

Comment:
Thanks @concretevitamin for the catch! Should command "more" work better than "cat" in this case? Or I should check for OS type.